### ISSUE TITLE
[Help] Error: no iamserviceaccounts match include glob filter specification

### ISSUE THREAD

**Issue by trombini77** [eksctl-io/eksctl#7046]
### What were you trying to accomplish?
I tried to using a eksctl yaml file to include after the EKS cluster was created, a ServiceAccount like this:

YAML BLOCK:

```
iam:
  withOIDC: true
  serviceAccounts:
    - metadata:
        name: Loki_S3
        # if no namespace is set, "default" will be used;
        # the namespace will be created if it doesn't exist already
        namespace: default
        labels: {aws-usage: "application"}
      attachPolicyARNs:
        - "arn:aws:iam::000000000:policy/Loki-Staging-01"
      tags:
        environment: "staging"
```
command executed: `eksctl create iamserviceaccount --config-file="eksctl_cluster_file.yaml" --include="Loki_S3"`

The documentation https://eksctl.io/usage/iamserviceaccounts/#usage-with-config-files says that command supports `--include` and `--exclude` flags.

### What happened?
```
Error: no iamserviceaccounts match include glob filter specification: "Loki_S3"
```

### How to reproduce it?

**Logs**

**Anything else we need to know?**

**Versions**

```
$ eksctl info
eksctl version: 0.153.0
kubectl version: v1.27.1-eks-2f008fe
OS: windows
```
and
```
$ eksctl info
eksctl version: 0.155.0
kubectl version: v1.27.1-eks-2f008fe
OS: windows
```

**Comment by TiberiuGC** [comment]
Hi @trombini77 , the `--include` `--exclude` flags do work with `create iamserviceaccount` indeed, albeit the docs could be a little more specific on how to use them. Point being, to uniquely identify a service account one needs both its name and namespace. Therefore, `eksctl` builds the name string for service accounts in the following format: `namespace/name` (see code snippets below).

https://github.com/eksctl-io/eksctl/blob/e34f235abbf69d953d5f392c4f09f89d7e3f9128/pkg/ctl/cmdutils/filter/iamserviceaccount_filter.go#L175-L181

https://github.com/eksctl-io/eksctl/blob/e34f235abbf69d953d5f392c4f09f89d7e3f9128/pkg/apis/eksctl.io/v1alpha5/iam.go#L119-L121

Therefore, your command should look like
```
eksctl create iamserviceaccount --config-file="eksctl_cluster_file.yaml" --include="default/Loki_S3"
```

I've also opened a PR adding this note to the docs.

**Comment by trombini77** [comment]
> Hi @trombini77 , the `--include` `--exclude` flags do work with `create iamserviceaccount` indeed, albeit the docs could be a little more specific on how to use them. Point being, to uniquely identify a service account one needs both its name and namespace. Therefore, `eksctl` builds the name string for service accounts in the following format: `namespace/name` (see code snippets below).
> 
> https://github.com/eksctl-io/eksctl/blob/e34f235abbf69d953d5f392c4f09f89d7e3f9128/pkg/ctl/cmdutils/filter/iamserviceaccount_filter.go#L175-L181
> 
> https://github.com/eksctl-io/eksctl/blob/e34f235abbf69d953d5f392c4f09f89d7e3f9128/pkg/apis/eksctl.io/v1alpha5/iam.go#L119-L121
> 
> Therefore, your command should look like
> 
> ```
> eksctl create iamserviceaccount --config-file="eksctl_cluster_file.yaml" --include="default/Loki_S3"
> ```
> 
> I've also opened a PR adding this note to the docs.

Great, now it is clear I´ll try with your explanation.

Thanks for adding it to the docs.

**Comment by trombini77** [comment]
After the merge of PR in docs, for us to see the update on https://eksctl.io/usage/iamserviceaccounts/#usage-with-config-files is necessary to wait some days?
I´m asking because at this moment that I´m writing down it continues without the new lines:

![image](https://github.com/eksctl-io/eksctl/assets/13336060/a1a9f273-8510-4411-b73c-4e6577c734d6)

**Comment by TiberiuGC** [comment]
@trombini77 yes, docs only get published when new eksctl releases are out. This change will probably be part of this week's release candidate `0.157.0-rc.0`. The doc change will be available when `0.157.0-rc.0` is promoted to a full release. You can check the corresponding release on the top right-hand corner on [eksctl.io ](https://eksctl.io/). 

<img width="486" alt="image" src="https://github.com/eksctl-io/eksctl/assets/110664232/567a0e4d-8d9c-40e8-aacf-3173eea644d9">
